DSharpPlus/Entities/DiscordMessage.cs

```diff
@@ -264,6 +264,13 @@ public Task<DiscordMessage> ModifyAsync(Optional<string> content = default, Opti
         public Task DeleteAsync(string reason = null) 
             => this.Discord.ApiClient.DeleteMessageAsync(this.ChannelId, this.Id, reason);
 
+        /// <summary>
+        /// Removes all embeds in the message.
+        /// </summary>
+        /// <returns></returns>
+        public Task SuppressEmbedsAsync()
```

Comment:
Since the payload takes a `true`/`false`, would be a good idea to reflect this as well (i.e. take a `bool` value with a default of true).